addons/website_slides/static/src/js/slides_share.js

```diff
@@ -115,7 +115,7 @@ publicWidget.registry.websiteSlidesShare = publicWidget.Widget.extend({
         var $clipboardBtn = $(ev.currentTarget);
         $clipboardBtn.tooltip({title: "Copied!", trigger: "manual", placement: "bottom"});
         var share_link_el = this.$('#wslides_share_link_id_' + $clipboardBtn[0].id.split('id_')[1]);
-        await browser.navigator.clipboard.writeText(share_link_el[0].innerText);    
+        await browser.navigator.clipboard.writeText(share_link_el[0].value);
```

Comment:
It seems safer to me to use `.val()` rather than `[0].value`: If there is no element selected, `[0].value` will raise an exception while `.val()` will return `undefined`.

Therefore, I would maybe do:

```suggestion
        await browser.navigator.clipboard.writeText(share_link_el.val() || '');
```

(same remark for the other suggested changes)

Quid if there is no text to copy? Do we still want to display the tooltip and update the clipboard?